TeslaLib/TeslaClient.cs

```diff
@@ -25,7 +25,7 @@ public class TeslaClient : ITeslaClient
         public string TeslaClientSecret { get; }
         public string AccessToken { get; private set; }
         // For refresh token.
-        private LoginToken _token;
+        private LoginToken loginToken;

```

Comment:
As a matter of style, one of the better .NET Framework design guidelines is that fields should start with a _ and method parameters should not, so you can easily differentiate them.  This looks like a search & replace gone wrong...

Reply:
Thanks. Done